app/javascript/dashboard/assets/scss/widgets/_buttons.scss

```diff
@@ -38,6 +38,10 @@
   &.round {
     border-radius: $space-larger;
   }
+
+  &.compact {
+    padding: 0;
```

Comment:
Not sure if putting 0 padding to left and right is a good idea, Can we at least put a small padding for left and right side ?